### ISSUE TITLE
CDF version is out of date

### ISSUE THREAD

**Issue by JonathonMSmith** [pysat/pysatCDF#34]
I feel obliged to mention that CDF is currently on version 3.8.1
It looks like pysatCDF is using CDF version 3.5.0 according to the README in cdf36_3-dist

I'm not sure which version is actually being used, but unless there are issues with the latest version I think we should try to use it.

**Comment by aburrell** [comment]
@JonathonMSmith agree!  I know this new version fixed some bugs.

**Comment by rstoneback** [comment]
Yep, now that pysatCDF actually builds again on modern systems I'm all for getting the new CDF code in.

**Comment by rstoneback** [comment]
My plan is to get the required changes I've identified in the meta workflow going and then while that is cooling and reviewing I'll come back here. The getting meta fixed when writing files, plus tests and proper docs will probably take the next couple days.
